typings/contract-deployment.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module 'contract-deployment' {
+    interface ContractBlockchainData {
+        abi: any;
+        query: any;
+        address: string;
+        bytecode: string;
+        defaultTxObject: any;
+        filters: any;
+        // TODO: Figure out how to specify in TypeScript that ContractBlockchainData can have any number of functions
```

Comment:
Ping me in dev with the details/example of what else goes on this interface and I can help figure out how to structure it.

Reply:
Since I have a TODO note here, let's revisit this later.  I have all of the other recommendations for this PR finished now, and I feel like this PR is about at a point where it's useable and can probably be merged.